### DISCUSSION TITLE
Quickdraw don't shown

### DISCUSSION THREAD

**Discussion by nagayev** [community/community#32152] [Other Feature Feedback, Questions, & Ideas]
I recently unlocked the Quickdraw achievement, but its not showing up with the other achievements on my profile, is this a bug?

**Comment by radar-blip** [comment]
This is not a known bug. Maybe you didn't do what is required, correctly. Keep Trying! 👍🏻

**Reply by nagayev** [reply]
@linuxlawson This is link to corresponding PR
https://github.com/nagayev/website/pull/20
It was merged about 4 minutes after opening.

**Reply by radar-blip** [reply]
I really think you should try again. Did you remember to close it within 5 mins also?

**Comment by Matteo-it** [comment]
Go on Settings > Public Profile and under Contributions & Activity check **Include private contributions on my profile**. Update preferences and you will see the badge!

**Comment by artiomtz** [comment]
Click the Achievements title on your profile page. If it's already in Achievements but not shown, open the achievement and toggle the eye icon for it to be shown on the profile page.

**Reply by mrudulmohan007** [reply]
iam not able to see achievements in my profile,could u guide me?

**Reply by radar-blip** [reply]
Here you go: https://github.com/mrudulmohan007?tab=achievements

**Comment by Jaber-Saed** [comment]
Firstly, congratulations on earning the Quickdraw achievement! It's definitely frustrating when your hard-earned accomplishments don't display as expected. Here are some steps you can take to address this situation:

Begin by refreshing the page. Sometimes, achievements might take a moment to sync up and appear on your profile. A simple page refresh could do the trick.

It's worth checking your privacy settings. Ensure that your profile settings are configured to showcase your achievements. On some platforms, you have control over the visibility of these accomplishments.

Logging out of your account and then logging back in could potentially resolve the issue. This action might trigger a synchronization process that brings the achievement to your profile.

Cached data or cookies in your browser might be causing the display problem. Consider clearing your browser's cache and cookies, and then revisit your profile to see if the achievement is now visible.

Should the problem persist, it might be time to get in touch with the platform's support team. Explain the situation in detail, including the name of the unlocked achievement and the issue you're facing. They should be able to assist you further.

Remember to stay updated with any recent changes or updates on the platform that might affect how achievements are displayed.

Engaging with the community can also be helpful. If the platform has a user forum or community, others might have encountered a similar issue and could provide insights or solutions.

Keep in mind that different platforms have unique systems for showcasing achievements, so the steps to address the issue might vary slightly. If you can provide more information about the platform you're using, I can offer more tailored guidance to help you resolve this matter.

**Comment by vanngoh** [comment]
### ATTENTION: For those who can't get the Quickdraw badge after merged the PR within 5 mins. 

You need to open an issue OR create a PR and then `close` it within 5 mins.
Note: Not `merge` the PR, it's `close`.

Hope this helps! Cheers!
